Formula/freeglut.rb

```diff
@@ -16,16 +18,34 @@ class Freeglut < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on :x11
+  depends_on "pkg-config" => :test
+  depends_on "libx11"
+  depends_on "libxi"
+  depends_on "libxrandr"
+  depends_on "libxxf86vm"
+  depends_on "mesa"
+
+  resource "init_error_func.c" do
+    url "https://raw.githubusercontent.com/dcnieho/FreeGLUT/c63102d06d09f8a9d4044fd107fbda2034bb30c6/freeglut/freeglut/progs/demos/init_error_func/init_error_func.c"
+    sha256 "74ff9c3f722043fc617807f19d3052440073b1cb5308626c1cefd6798a284613"
+  end
 
   patch :DATA
 
   def install
     inreplace "src/x11/fg_main_x11.c", "CLOCK_MONOTONIC", "UNDEFINED_GIBBERISH" if MacOS.version < :sierra
+    inreplace "CMakeLists.txt", "/opt/X11", Formula["mesa"].opt_prefix.to_s
```

Comment:
Can this be submitted upstream somehow?

Reply:
It's really a cmake issue, see my comment below.

Reply:
Thanks to @jonchang, that's no longer needed.